config/install/field.storage.node.field_node_primary_site.yml

```diff
@@ -0,0 +1,19 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - node
+    - taxonomy
+id: node.field_node_primary_site
+field_name: field_node_primary_site
+entity_type: node
+type: entity_reference
+settings:
+  target_type: taxonomy_term
+module: core
+locked: false
+cardinality: 1
+translatable: true
+indexes: {  }
+persist_with_no_fields: false
+custom_storage: false
```

Comment:
hey @asarfara , tide_core doesn't have dependency with tide_site. so this file doesn't need to be added.